quote-app/resources/views/dashboard.blade.php

```diff
@@ -9,7 +9,14 @@
         <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
             <div class="bg-white overflow-hidden shadow-sm sm:rounded-lg">
                 <div class="p-6 bg-white border-b border-gray-200">
-                    You're logged in!
+                    <div class="flex flex-row justify-center ">
+                        @foreach($quotes as $quote)
+                            <div class="bg-emerald-200" style="width:24%; height:100px; text-align:center; margin:1%;" >
```

Comment:
let's remove inline css and let's use tailwind classes here:
let's do these changes:
width:24% => w-1/4
height:100px => h-28
text-align:center; => text-center
margin:1%; => find the best class here: https://tailwindcss.com/docs/margin